### PR TITLE
added about.html

### PR DESCRIPTION
**Pull request by tejaswini22199**
issue #2  
resolved issue 2 by adding about.html

### CODE REVIEW

File: bloggitt/core/urls.py [hunk 1]
```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
     path('', views.postlist, name='home'),
     path('<slug:slug>/', views.postdetail, name='post_detail'),
+    path('/',views.aboutdetail,name='about'),
```
Comment (diyajaiswal11) [inline bloggitt/core/urls.py:7]: The path should be 'about/'. Make the changes.

**Comment by tejaswini22199** [comment]
hey I added but do I need to create another pull request

**Comment by diyajaiswal11** [comment]
> hey I added but do I need to create another pull request

No @tejaswini22199 , you don't have to create another pull request. Also, have you registered as a student at NWoC'20?

**Comment by tejaswini22199** [comment]
> > hey I added but do I need to create another pull request
> 
> No @tejaswini22199 , you don't have to create another pull request. Also, have you registered as a student at NWoC'20?

yes I have registered.

**Comment by diyajaiswal11** [comment]
> > > hey I added but do I need to create another pull request
> > 
> > 
> > No @tejaswini22199 , you don't have to create another pull request. Also, have you registered as a student at NWoC'20?
> 
> yes I have registered.

Okay @tejaswini22199 , I'll merge your pull request after 1st December, so that it's counted for NWoC'20.

**Pull request merged**